inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp

```diff
@@ -127,6 +127,8 @@ class LayerTestsCommon : public CommonTestUtils::TestsCommon {
 
     virtual void ConfigureNetwork();
 
+    virtual void ConfigureNetwork_Secondary() {}
```

Comment:
I didn't catch why the function is needed. Moreover looks strange and confusing. Remove it

Reply:
It's a room for secondary input shape for a kind of eltwise requested by Maxim. Then do you mean eltwise style input needs not to be supported?

Reply:
I didn't catch how is it connected with multiinput?

Reply:
It's for eltwise style input, ConfigureNetwork() is used for one input case, and ConfigureNetwork_Secondary() is used for two input case if it is needed. Actually I'm not sure if eltwise supports two input case, but Maxim's code uses two input case only.